core/templates/pages/contributor-dashboard-page/modal-templates/translation-modal.component.html

```diff
@@ -153,6 +153,20 @@
     </span>
   </button>
 </div>
+<div *ngIf="modifyTranslationOpportunity && !opportunity" class="oppia-translate-modal-footer modal-footer">
```

Comment:
We don't need another cancel button per say, it is because we need another save button and thus a separate footer div for differentiation depending on whether it's being used from CD or the exploration editor page